notebooks/notebooks/paper/defense-win-rate-heatmap.ipynb

```diff
@@ -10,7 +10,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.    full_victim_visits = ["4", "16", "256", "4096"]
</pre><p>these constants got deleted the last time this file was modified, somehow I didn't notice</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlignmentResearch/KataGoVisualizer/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>